src/test/java/org/elasticsearch/test/rest/ElasticsearchRestTestCase.java

```diff
@@ -302,4 +321,86 @@ public void test() throws IOException {
             executableSection.execute(restTestExecutionContext);
         }
     }
+
+    // don't look any further: NO TOUCHY!
+    
+    public static class Rest0Tests extends ElasticsearchRestTestCase {
```

Comment:
this is done to workaround reporting problems when the same test runs on multiple jvms but a single joint report is generated right? maybe add a better to just explain this?